website/content/docs/secrets/aws.mdx

```diff
@@ -95,8 +83,17 @@ management tool.
     on using STS Federation Tokens, since the temporary security credentials
     associated with the role are not authorized to use GetFederationToken.)
 
+    In some cases, providing sensitive IAM security credentials to Vault as configuration 
+    is not possible. For example, your organization may require that all security 
+    credentials are short-lived or tied to a machine identity. We recommend using Vault 
+    [plugin workload identity federation](#plugin-workload-identity-federation-wif) (WIF) 
+    in this case.
+
+    See the IAM [permissions policy](#iam-permissions-policy-for-vault) for Vault section 
+    for the permissions the credentials need to have for Vault to perform its actions.
```

Comment:
```suggestion
    In some cases, you cannot set sensitive IAM security credentials in your
    Vault configuration. For example, your organization may require that all
    security credentials are short-lived or explicitly tied to a machine identity.
    
    To provide IAM security credentials outside your Vault configuration, we
    recommend using Vault
    [plugin workload identity federation](#plugin-workload-identity-federation-wif)
    (WIF).

    Refer to the IAM [permissions policy for Vault](#iam-permissions-policy-for-vault)
    section for details on the permission requirements for using WIF.
```

Reply:
> To provide IAM security credentials outside your Vault configuration

There aren't IAM security credentials being provided outside of the Vault configuration. They aren't being provided at all in the case of plugin WIF.

> Refer to the IAM [permissions policy for Vault](#iam-permissions-policy-for-vault) section for details on the permission requirements for using WIF.

These are the general permissions needed regardless of using plugin WIF or not.

Happy to commit a different suggestion or commit this one followed by edits!

Reply:
Tweaked the suggestion based on your clarifications. Given that the permissions are general, and not WIF-specific, I don't think it adds much to call them out here. If they aren't being highlighted effectively elsewhere in the doc, we can add them as a suggested reading at the end

Reply:
Makes sense, thanks!